src/main/java/duke/commands/DeleteCommand.java

```diff
@@ -0,0 +1,51 @@
+package duke.commands;
+
+import duke.exception.DukeException;
+import duke.main.Storage;
+import duke.main.Ui;
+import duke.tasks.Task;
+import duke.tasks.TaskList;
+
+/**
+ * DeleteCommand used to delete tasks
+ */
+public class DeleteCommand extends Command {
+
+    private final int index;
+
+    /**
+     * Constuctor for DeleteCommand
+     *
+     * @param description String representation of task number to be marked
+     * @throws DukeException if user did not type in a correct task number
+     */
+    public DeleteCommand(String description) throws DukeException {
```

Comment:
I like the naming convention, clearly tells me what the method does without having to read the implementation.